docs/chart-design-and-style/style-by-css.md

```diff
@@ -11,15 +11,33 @@ To customize your styles, you can create your own themes, or just add your own i
 
 Highcharts since v11 honors the [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) CSS media feature. If you end user prefers dark mode, the Highcharts CSS will pick this up and set the CSS color variables accordingly. To avoid this, you can set either the `.highcharts-light` or `.highcharts-dark` class name on the chart container. See a [live demo for switching between modes](https://www.highcharts.com/samples/highcharts/css/prefers-color-scheme).
 
+## Loading from node_modules
```

Comment:
This header comes out stronger than the simpler option, which is loading from CDN. The CDN is not mentioned specifically, but linked to in the intro. Reorganize a bit? Maybe a header `## Loading the CSS` then with two sub-headers `Loading from CDN` and `Loading from node_modules`?